ETC.tsv

```diff
@@ -13250,7 +13250,7 @@ ETC_20150717_013250	Use {img N 40 40} key to open World Map.
 ETC_20150717_013251	Areas you haven't been to will not be shown in the World Map,{nl}while areas you have already been to will be activated.
 ETC_20150717_013252	Moving your mouse over the activated area will show the map that the character has completed.
 ETC_20150717_013253	You can check your items, equipments, money, and weight through the Inventory.
-ETC_20150717_013254	Use {img F2 40 40}키 key to open Inventory window.
+ETC_20150717_013254	Use {img F2 40 40} key to open Inventory window.
```

Comment:
Use {img F2 40 40} key to open Inventory window. **->** Press {img F2 40 40} to open the Inventory window.

For consistency.